source/endpoints/devices/Device.py

```diff
@@ -1,9 +1,8 @@
 from flask_restplus import Resource
 from flask_restplus import fields
 
-from endpoints.devices import namespace
+from endpoints.devices import namespace, business_logic_device
```

Comment:
I think that the business logic should be in a seperate module and not in he endpoint module